core/src/i18n/en.yaml

```diff
@@ -1365,6 +1366,27 @@ oauth2-clients:
     secret-name: Secret name points to the Kubernetes Secret that contains the client's ID and password.
 observability:
   title: Observability
+persistent-volume-claims:
+  description: <0>Persistent Volue Claim</0> is a request for the Persistent Volume resources.
+  headers:
+    access-modes: Access Modes
+    configuration: Configuration
+    key: Key
+    match-expressions: Match expressions
+    match-labels: Match Labels
+    operator: Operator
+    selector: Selector
+    storage: Storage
+    storage-class: Storage Class
+    values: Values
+    volume-mode: Volume Mode
+    volume-name: Volume Name
+  name_singular: Persistent Volume Claim
+  title: Persistent Volume Claims
+  tooltips:
+    bound: Binding between Persistent Volume and Persistent Volume Claim established
+    lost: Lost binding between Persistent Volume and Persistent Volume Claim
```

Comment:
Maybe it would make sense to adjust the status to the most common ones and change `lost` to `error`, and just keep this tooltip? 
(to do so, create the same entry, in the `statuses` object like [here](https://github.com/kyma-project/busola/blob/main/core/src/i18n/en.yaml#L931) for example )

Reply:
If we want to keep the same naming convention as k8s does, the statuses should stay like that. PVC can have [three](https://www.humblec.com/persistent-volume-and-claim-pv-and-pvc-status-in-kubernetes/) various statuses